lib/merge_hris_client/configuration.rb

```diff
@@ -218,7 +220,7 @@ def auth_settings
             type: 'api_key',
             in: 'header',
             key: 'Authorization',
-            value: api_key_with_prefix('Authorization')
+            value: api_key_with_prefix('tokenAuth')
```

Comment:
we really shouldn't change the way this sdk handles auth, that's a pretty unnecessary breaking change

Reply:
Was the prior version of this SDK generated with non-5.1.1 of openapi generator or something? I didn't tweak that at all.